src/translations/en.json

```diff
@@ -936,6 +936,19 @@
             "required_fields": "Fill in all required fields"
           }
         }
+      },
+      "lovelace": {
+        "editor": {
+          "header": "Card Configuration",
```

Comment:
Feels weird that one dialog is "top level" of editor and one below it. I think both should be below it: `edit.header` and `migrate.header`